code/server/Scripting/Scripting.cpp

```diff
@@ -175,15 +175,22 @@ void Scripting::Initialize()
     spdlog::get("Scripting")->debug("Done loading plugins.");
 }
 
-component_entry_point_fn Scripting::GetFunction(const wchar_t* apName) const
+component_entry_point_fn Scripting::GetFunction(const wchar_t* apNameW) const
 {
+#if TP_PLATFORM_WINDOWS
+    const wchar_t* apName = apNameW;
```

Comment:
this is not ok, variable names start with a when they are arguments

Reply:
Aah, okay, thank you! I didn't realize the convention